### ISSUE TITLE
cargo-checksum.sh need execute(+x) rights

### ISSUE THREAD

**Issue by VictorLux** [z-classic/zclassic#192]
On Linux, you need to perform this command in order to build without errors. chmod +x ......../zclassic-1.1.1/depends/cargo-checksum.sh

**Comment by nimbosa** [comment]
yes, we are getting more and more of these permission errors, we will add documentation to make this part of the build process, without changing the git repo, maybe like so:

```
git config --global core.filemode false
git clone https://github.com/z-classic/zclassic.git
cd zclassic
git checkout dev
git config core.filemode false
sudo chown $(whoami) -R .
sudo chown $(whoami) -R ./*
sudo chmod a+x -R .
sudo chmod a+x -R ./*

### CHECK:
git status
```

we can fix this at the same time as https://github.com/z-classic/zclassic/issues/188

**Comment by nimbosa** [comment]
Like other works in progress here, this is overdue. We will prioritize "low-hanging fruits" and close issues on collab/communication, documentation and instruction first.
